.buildkite/pipelines/pipeline.kibana-release.yaml

```
@@ -0,0 +1,9 @@
+steps:
+  - label: ":releasethekraken: Release kibana"
+    # https://regex101.com/r/tY52jo/1
+    if: 'build.env("BUILDKITE_TAG") =~ "/^deploy@\d+$$/"'
```

Tiny nit/curiosity question: From [this similar pipeline](https://github.com/elastic/apm-index-service/blob/cb105559d2480d64c61f8edfc736fd7c805d537a/.buildkite/pipeline.yml#L30), I can this has been done slightly differently previously. But I guess the two are basically doing the same?

```suggestion
    if: build.tag =~ /^deploy@\d+\$/
```

This is also similar to what you've done in `catalog-info.yaml` below.

(btw, is `$$` a way to escape the `$` in this file?)

---

you're probably right, `build.tag` is listed to be the tag the build is based on: https://buildkite.com/docs/pipelines/conditionals

---

I also updated the other regex to use `\` vs `$` for escaping, since `\` is preferred and more obvious escape sequence